dymos/examples/min_time_climb/test/test_min_time_climb_simulate_failure.py

```diff
@@ -95,9 +95,9 @@ def test_simulate_raises_analysis_error(self):
 
         with self.assertRaises(om.AnalysisError) as e:
             dm.run_problem(p, run_driver=False, simulate=True)
-        self.assertEqual(str(e.exception), "simulation of traj.phases.phase0.segments.segment_2 "
-                                           "failed: Required step size is less than spacing "
-                                           "between numbers.")
+            self.assertEqual(str(e.exception), "simulation of traj.phases.phase0.segments.segment_2 "
```

Comment:
Isn't this the same thing as removing the assert that checks the error message? Once the analysis error is raised during `run_problem`, you leave the context block and never execute that line.

Reply:
Yes, that is a problem. I thought it is also a problem that "e" is not in scope after the "with" returns, but hat does not seem to be the case. I will fix the assertEqual text.

Reply:
I misunderstood "with" scope in Python. The named variable is available until the end of the function.

Reply:
yeah python scoping is odd for with blocks